hunt/app/views/animals/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Animals#show</h1>
-<p>Find me in app/views/animals/show.html.erb</p>
+
+<%= link_to 'View All Animals', animals_path, class: 'btn btn-dark' %>
```

Comment:
Can you move this link inside the column as well? After that we should be good to merge! 🎉